src/scripting.cc

```diff
@@ -74,12 +74,10 @@ enum {
   LL_WARNING,
 };
 
-extern "C" {
```

Comment:
The tcl test cases failure seems was caused by this change.

Reply:
Sorry, the error is caused by a typo in cmake, I have fixed it in the latest commit.
If `extern "C"` is added, the linker will fail, since we compile lua by c++ compiler and so these lua symbols is mangled by Itanium C++ ABI.

Reply:
`extern "C"` in C++ is basically to prevent function symbols from mangling by c++ abi.